### ISSUE TITLE
Duplicated shortcuts

### ISSUE THREAD

**Issue by aaaaaadrian** [BrowserWorks/waterfox#3501]
### What happened?

As copied from https://github.com/BrowserWorks/Waterfox/issues/3491

In Firefox and Floorp, using ⇧+⌥+⬅️ or ⇧+⌥+➡️ would allow me to select whole words at a time starting from the cursor, instead of using ⇧+⌘+⬅️ or ⇧+⌘+➡️ which selects the entire line from the cursor position.

In Waterfox, using ⇧+⌥+⬅️ or ⇧+⌥+➡️ cycles through open tabs, which is a duplicated function of ⇧+⌘+[ or ⇧+⌘+], respectively.

The ability to disable the ⇧+⌥+⬅️ or ⇧+⌥+➡️ shortcuts for tab browsing, or at least be able to choose either ⇧+⌥+⬅️ / ⇧+⌥+➡️ or ⇧+⌘+[ / ⇧+⌘+] is necessary

I have searched the advanced config and thought about extensions for shortcuts, but I don't want to bloat the browser.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

G6

### What platform are you seeing the problem on?

macOS

### Relevant log output

```shell
G6.0.17 (64-bit)
```

**Comment by hannes-ucsc** [comment]
Same here. This bug was introduced recently. In text fields, I frequently use Option-Shift-Left and Option-Shift-Right to select the word before or after the cursor. This is driving me nuts. It happened twice while writing this. :-(

**Comment by hannes-ucsc** [comment]
I cannot reproduce this in the latest Firefox.

**Comment by hannes-ucsc** [comment]
Also, I think the title of he issue may be misleading. The issue is not so much that one function has two shortcuts, it is the fact that an OS-default keyboard shortcut for text editing (select word left/right) is overridden and bound to another function (switch tabs). It would be OK if (Option-Shift-Left/Right) remained bound to the switch tabs function, as long as no text editing field has the focus.

**Comment by hannes-ucsc** [comment]
This is reproducible in 6.0.17 and 6.0.16 but not in 6.0.15.

**Comment by hannes-ucsc** [comment]
OK, I don't claim that my experience is representative of that of all or even many users, but I can't use Waterfox with this bug. Muscle memory has me type Option-Shift-Left/Right every ten seconds. I had to revert to 6.0.15 and am now stuck without security updates. I will have to move to a different browser if this isn't addressed. Not complaining, just describing my situation.

**Comment by glass-shard** [comment]
I'm also experiencing this. I've recently attempted to switch to Waterfox and would love to make it my main browser over Firefox, but this bug kills it for me.

**Comment by Foxsly** [comment]
I've been running into this quite a bit recently too. Pretty disconcerting to be expecting some text to get highlighted and get switched to another tab instead.

**Comment by natebeaty** [comment]
Another potential Mac Waterfox user that uses shift-alt-arrow keys *all* the time in text areas (often while editing URLs to quickly change paths while developing), making Waterfox completely unusable in its current state. I had this happen several times while typing this issue out. I was hoping the latest update would fix this, but seems to still be happening.

**Comment by oubiwann** [comment]
Yeah, last month I downgraded and pinned my main machine to WF 6.0.15. I was hoping to see this fixed by now, but today I hit my limit on my laptop, and am currently downgrading and pinning that one, too :-((((

**Comment by hannes-ucsc** [comment]
I switched to LibreWolf which is also a PITA, but in less annoying ways. I just need a browser with container tabs and reasonable privacy settings.

**Comment by acress1** [comment]
Hi,
It would be awesome to fix it.
To be more precise, in the devtools it works, it doesn't go from one tab to another.
It doesn't work in the url bar, in the search bar, and also in the windows...

**Comment by natebeaty** [comment]
Looks like this is now fixed. Thank you to whoever squashed this!
